packages/office-ui-fabric-react/src/components/HoverCard/examples/HoverCard.Basic.Example.tsx

```diff
@@ -1,44 +1,49 @@
+// @codepen
+
 import * as React from 'react';
-import { BaseComponent } from 'office-ui-fabric-react/lib/Utilities';
 import { HoverCard, IExpandingCardProps } from 'office-ui-fabric-react/lib/HoverCard';
+import { Fabric } from 'office-ui-fabric-react/lib/Fabric';
 import { DetailsList, buildColumns, IColumn } from 'office-ui-fabric-react/lib/DetailsList';
-import { createListItems } from 'office-ui-fabric-react/lib/utilities/exampleData';
-import './HoverCard.Example.scss';
-
-let _items: any[];
-
-export interface IHoverCardExampleState {
-  items?: any[];
-  columns?: IColumn[];
-}
+import { createListItems, IExampleItem } from 'office-ui-fabric-react/lib/utilities/exampleData';
+import { mergeStyleSets } from 'office-ui-fabric-react/lib/Styling';
 
-export class HoverCardBasicExample extends BaseComponent<{}, IHoverCardExampleState> {
-  constructor(props: {}) {
-    super(props);
-
-    _items = _items || createListItems(10);
-
-    this.state = {
-      items: _items,
-      columns: _buildColumns()
-    };
+const classNames = mergeStyleSets({
+  compactCard: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    height: '100%'
+  },
+  expandedCard: {
+    padding: '16px 24px'
+  },
+  item: {
+    selectors: {
+      '&:hover': {
+        textDecoration: 'underline',
+        cursor: 'pointer'
+      }
+    }
   }
+});
 
-  public render() {
-    const { items, columns } = this.state;
+export class HoverCardBasicExample extends React.Component<{}, {}> {
+  private _items: IExampleItem[] = createListItems(10);
+  private _columns: IColumn[] = this._buildColumns();
 
+  public render() {
     return (
-      <div>
+      <Fabric>
```

Comment:
Just curious... why changing this `div` to `Fabric`?

Reply:
So that the text inside it will be formatted with the proper fonts when exported to codepen.